### PR TITLE
Make service token persistent on disk

### PR DESCRIPTION
**Pull request by emanuelen5**

### CODE REVIEW

File: login.py [hunk 1]
```diff
@@ -78,14 +40,14 @@ def main():
 
     for s in services:
         if s == "slack":
-            token_path = os.path.join(ns.ramdisk_path, config.slack_token_filename)
+            token_path = config.slack_token_filename
             if ns.slack_channel_id is None:
                 logger.error("The Slack channel ID must be specified")
                 print("Skipping Slack login")
                 continue
```
Comment (InMyOrbit) [inline login.py:47]: This is not executed when program is started without --slack-channel-id

Comment (emanuelen5) [reply login.py:47]: Isn't that intentional? You need both a Slack channel ID and a token to be able to connect to Slack.

File: memberbooth.py [hunk 2]
```diff
@@ -63,18 +61,16 @@ def main():
...
+                                                   token_path=config.makeradmin_token_filename)
     else:
-        makeradmin_client = MakerAdminClient(base_url=ns.maker_admin_base_url, token_path=makeradmin_token_path)
+        makeradmin_client = MakerAdminClient(base_url=ns.maker_admin_base_url, token_path=config.makeradmin_token_filename)
 
-    slack_token_path = os.path.join(ns.ramdisk_path, config.slack_token_filename)
     if no_slack:
-        slack_client = MockSlackClient(token_path=slack_token_path, channel_id=ns.slack_channel_id)
+        slack_client = MockSlackClient(token_path=config.slack_token_filename, channel_id=ns.slack_channel_id)
     else:
```
Comment (InMyOrbit) [inline memberbooth.py:72]:
```suggestion
    else:
        if ns.slack_channel_id is None:
            logger.error(f"Missing --slack-channel-id.")
            sys.exit(-1)
```

**Review by InMyOrbit — CHANGES_REQUESTED** [review]
Add check for missing --slack-channel-id in member booth.py, suggestion below.

**Pull request merged**